.travis.yml

```diff
@@ -1,19 +1,58 @@
 dist: trusty
 language: java
-jdk: oraclejdk8
+jdk: openjdk8
 
 addons:
   apt:
     packages:
       - xvfb
 
-install:
-  - export DISPLAY=':99.0'
-  - Xvfb :99 -screen 0 1024x768x24 > /dev/null 2>&1 &
+x-test-install: &test-pre-install
+  - wget -O ~/install-jdk.sh https://github.com/sormuras/bach/raw/8ca3ad9/install-jdk.sh
+  - chmod +x ~/install-jdk.sh
+  - &x-setup
+    export DISPLAY=':99.0'; Xvfb :99 -screen 0 1024x768x24 > /dev/null 2>&1 &
 
 cache:
   directories:
     - $HOME/.m2
 
-script:
-  - mvn clean verify -e
+jobs:
+  include:
+    - stage: Verify on J8
+      name: OpenJDK8
+      before_install:
+        - *x-setup
+      script:
+        - find > list-before
+        - mvn -T1C clean verify -e -fae
+        - |
+          for tgt in $TRAVIS_BUILD_DIR/*/target/; do
+            module=${tgt%/target/}
+            mkdir -p ~/.m2/artifact-temp/"$module"
```

Comment:
`~/.m2/artifact-temp/` appears multiple times in this yaml. Can you move it to a variable